dataprofiler/profilers/profile_builder.py

```diff
@@ -19,6 +19,8 @@
 import pandas as pd
 import numpy as np
 
+import pickle
```

Comment:
should be grouped with the built-in imports above, can just go on line 16 so no space between warnings and mp

Reply:
Fixed in new commit